CONTRIBUTING.md

```diff
@@ -20,6 +20,11 @@ Contributions are welcome! Here's how you can help:
     - Check your code works as expected and document any changes to the Lua API.
 
 4. Commit & [push](https://help.github.com/articles/pushing-to-a-remote/) your changes to a new branch (not `master`, one change per branch)
+    - Commits should:
+        - Be short and sweet. Large commits are hard to review and reason
+          about. If your commit grows too large, consider splitting it into
+          smaller pieces.
+        - Focus on one thing only.
```

Comment:
Wording doesn't seem right.

Reply:
Surely the focus should be more on the PR than individual commits within the PR. 

PRs shouldn't always be short. A PR should contain as much content as is required for a distinct and specific change or addition, and for larger changes contributors should consider splitting their work into multiple PRs when there are no disadvantages and where doing so may simplify the review process.

It's somewhat difficult to sum up precisely because there will always be judgement calls involved in this. It would be ridiculous to submit multiple PRs for distinct spelling changes in different sentences, so sometimes grouping similar changes is good, while other times things are just easier when separated.